client/webserver/http.go

```diff
@@ -260,13 +260,15 @@ func (s *WebServer) handleSettings(w http.ResponseWriter, r *http.Request) {
 		FiatCurrency    string
 		Exchanges       map[string]*core.Exchange
 		IsInitialized   bool
+		AppVersion      string
 	}{
 		CommonArguments: *common,
 		KnownExchanges:  s.knownUnregisteredExchanges(xcs),
 		FiatCurrency:    core.DefaultFiatCurrency,
 		FiatRateSources: s.core.FiatRateSources(),
 		Exchanges:       xcs,
 		IsInitialized:   s.core.IsInitialized(),
+		AppVersion:      s.appVersion,
```

Comment:
I don't guess there's a reason to pass this around? Any of the golang code we can just import `client/app` and use `app.Version`? Can just do that here?

Reply:
Thanks .. I will change.

Reply:
> I don't guess there's a reason to pass this around? Any of the golang code we can just import `client/app` and use `app.Version`? Can just do that here?

Here cannot as conflicting imports for `client/app` in `webserver_test.go` but I have reworked to pass `app.Version`

into:

`webserver.New(cfg.Web(clientCore, marketMaker, logMaker.Logger("WEB"), utc, app.Version))` 

in:

`client/cmd/bisonw/main.go`